app/javascript/spec/select/select.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import FormRenderer from '@data-driven-forms/react-form-renderer';
+import { FormTemplate, componentMapper } from '@data-driven-forms/pf3-component-mapper';
+import Select from '../../components/select';
+
+const RendererWrapper = ({ onChange }) => (
+  <FormRenderer
+    onSubmit={ () => {} }
+    FormTemplate={ FormTemplate }
+    componentMapper={{
+      ...componentMapper,
```

Comment:
You're not using any of the components from `componentMapper` so it's safe to delete it and you also don't need its import above.